dataactcore/models/baseInterface.py

```diff
@@ -161,8 +144,32 @@ def getNameFromDict(self, model, dictName, fieldName, fieldValue, idField):
         raise ValueError("Value: " + str(fieldValue) + " not found in dict: " + str(dict))
 
 
+def dbConnection():
+    """Use the config to set up a database engine and connection"""
+    if not CONFIG_DB:
+        raise ValueError("Database configuration is not defined")
+
+    dbName = CONFIG_DB['db_name']
```

Comment:
Ah, I see, this handles the case where `dbName` is in the config file but doesn't have a value. I'm with you now..yeah, we can leave this, though I'm generally in favor of explicitly checking for things that could go awry in a user-configured file.

If I was doing this again,does it make sense to do a `.get` so people get a